package.json

```diff
@@ -4,10 +4,11 @@
   "description": "x11 javascript hashing algorithm in pure javascript",
   "main": "index.js",
   "scripts": {
-    "test": "mocha test/test.js",
+    "test": "mocha --timeout 10000 --full-trace --check-leaks test/test.js",
     "clean": "rm -rfv dist/*",
     "benchmark": "node test/benchmarks.js",
-    "build": "grunt"
+    "lint": "node_modules/.bin/eslint lib",
```

Comment:
Why not just `eslint lib`? We shouldn't need to specify path with `node_modules/.bin/` in the scripts...

Reply:
fixed